platform/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+services:
+  postgres:
+    image: bitnamisecure/postgresql:latest
```

Comment:
bitnami is sunsetting their repo, could you please use postgres like in https://github.com/archestra-ai/website/blob/main/app/docker-compose.yml#L1